bcipy/parameters/parameters.json

```diff
@@ -630,14 +646,54 @@
     "recommended_values": "",
     "type": "bool"
   },
+  "rectangle_message_box": {
+    "value": "false",
+    "section": "feedback_config",
+    "readableName": "rectangle_message_box",
+    "helpTip": "Show rectangular shape around feedback.",
+    "recommended_values": "",
+    "type": "bool"
+  },
+  "circular_message_box": {
+    "value": "false",
+    "section": "feedback_config",
+    "readableName": "circular_message_box",
+    "helpTip": "Show circular shape around feedback.",
+    "recommended_values": "",
+    "type": "bool"
+  },
   "feedback_message_color": {
-    "value": "green",
+    "value": "white",
     "section": "feedback_config",
     "readableName": "feedback_message_color",
     "helpTip": "feedback_message_color",
     "recommended_values": "",
     "type": "str"
   },
+  "feedback_line_color": {
+    "value": "white",
+    "section": "feedback_config",
+    "readableName": "feedback_line_color",
+    "helpTip": "feedback_line_color",
+    "recommended_values": "",
+    "type": "str"
+  },
+  "feedback_padding": {
+    "value": "0.37",
+    "section": "feedback_config",
+    "readableName": "feedback_padding",
+    "helpTip": "feedback_padding",
+    "recommended_values": "",
+    "type": "float"
+  },
+  "feedback_target_line_width": {
+    "value": "100",
```

Comment:
I tried this locally and 100 seemed way too wide; maybe 10?

Reply:
Never mind; this seems very display-specific and 100 was too narrow during testing (I believe 150 worked better). So probably just leave this as is for now.